layouts/_default/impact.html

```diff
@@ -0,0 +1,83 @@
+{{ define "main" }}
```

Comment:
This is mostly the same as the list layout, right? I'd be interested in seeing if we could switch this to just using blocks and deleting impact.html as a follow up PR later. It doesn't need to be done now though.